content/tutorials/install-grafana-on-raspberry-pi.md

```diff
@@ -16,7 +16,7 @@ Many people are running Grafana on Raspberry Pi as a way to monitor their home,
 
 In this tutorial, you'll:
 
-- Set up a headless Raspberry Pi using Raspbian.
+- Set up a headless Raspberry Pi using Raspberry Pi OS (previously known as Raspbian).
```

Comment:
Is headless really the best way to describe this?

Reply:
It's a good SEO word and is an accurate description, but I can expand on it for newbies. What do you think of this?
Set up a Raspberry Pi using a version of Raspberry Pi OS that does not require you to connect a keyboard or monitor (this is often called "headless").

Reply:
Long, but I think it's better and way more informative!